modules/ROOT/pages/04-scaling.adoc

```diff
@@ -219,7 +219,7 @@ The autoscale pods is computed using the formula:
 totalPodsToScale = inflightRequests / concurrencyTarget
 ----
 
-With this current setting of **concurrencyTarget=10**(`containerConcurrency: "10"`) and **inflightRequests=50** , you will see Knative automatically scales the greeter services to  `**50/10 = 5 pods**`.
```

Comment:
I think knative is mostly moving away from containerConcurrency because it doesn't work well and doesn't allow service to properly scale. So I'd not mention it here. It's just confusing in this example, IMO.

Reply:
makes sense